### ISSUE TITLE
The custom format helpers `createPropertyFormatter` does not support passing formatting options

### ISSUE THREAD

**Issue by yangchch6** [amzn/style-dictionary#941]
In the `createPropertyFormatter` function, I can't pass in custom formatting attributes, because it will be overridden by hard-coded `prefix='--'`

![image](https://user-images.githubusercontent.com/33412781/220063058-aa032236-313a-4c21-95fd-9564f7cec967.png)

This code should be changed like this, could you support the fix of this problem?
![image](https://user-images.githubusercontent.com/33412781/220063558-354ce322-357e-4fa5-86d8-283c44f99d68.png)

**Comment by frshwtr** [comment]
Hello.

I have found the way to get around this is to not set the `format` property. You will then need to define all the other formatting styles you need though.

Bumping this as I would like to be able to use a prefix with `format` also being set. In my project, if i remove `format` to get the desired prefix, then I lose the behaviour of `outputReferences`. I require both for my use case.

**Comment by torytang025** [comment]
I had the same problem. It seems like a bug 😢 
For me, I copied this file and rewrote it like your proposal

**Comment by jorenbroekema** [comment]
https://github.com/amzn/style-dictionary/pull/1165 once this is merged and released in v4.0.0-prerelease.27, you will be able to do it like so:

```js
{
  destination: 'variables.css',
  format: 'css/variables',
  options: {
    formatting: { 
      indentation: '    ',
    },
  },
}
```
indentation just being one example but the other props work as well such as prefix etc.
